spec/requests/tutorials/show_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe TutorialsController, type: :request do
+  describe 'GET #show' do
+    let! :tutorial do
+      create(
+        :tutorial,
+        title: 'Test Tutorial Title'
```

Comment:
This uses the `tutorial` factory and specifically overrides its `title` (since our default is to use a `sequence`).

When the specs run, the database doesn't get cleaned out between specs, so this spec was failing with different titles: `Test Tutorial Title 2` on one run, `Test Tutorial Title 6` on another, etc. etc.

Specs are run in random order, so there wasn't an easy way to predict _which_ title would end up being run in our spec. That's a good thing, but not what we want here, obviously. So… rather than implement something like [Database Cleaner](https://github.com/DatabaseCleaner/database_cleaner), I simply overwrite the `title` that's used.